modules/docs/markdown/01-overview/05-sharing-specs.md

```diff
@@ -200,14 +200,21 @@ will result in the `"software.amazon.smithy" % "smithy-aws-iam-traits" % "1.14.1
 and used for the smithy-level classpath of the smithy files contained by `downstream`. This effectively means that smithy files
 in `downstream` can use the Smithy shapes present in the `smithy-aws-iam-traits` artifact.
 
-
-
-One side-effect of this is that if you produce JARs containing artifacts produced by Smithy4s code generation, they'll contain a `smithy4s.tracking.smithy`. This could be a problematic file if you're using `sbt-assembly` because if you depend on multiple JARs that contain this file, you'll need to write a custom `assemblyMergeStrategy`, like so:
+One side-effect of this is that if you produce JARs containing artifacts produced by Smithy4s code generation, they'll contain some common files with other smithy projects, particularly within the `META-INF` folder. You'll need to write a custom `assemblyMergeStrategy`, like so:
 
 ```sbt
 assemblyMergeStrategy := {
-  case "META-INF/smithy/smithy4s.tracking.smithy" =>
-    MergeStrategy.discard
+  case PathList("META-INF", xs @ _*) =>
+        (xs map { _.toLowerCase }) match {
```

Comment:
I think these parens are redundant

```suggestion
        xs map { _.toLowerCase } match {
```

also, there's a bit too much nesting in these new lines

Reply:
Committed the suggestion. WDYM by too much nesting? Happy to update as needed.

Reply:
right, I meant indentation 😅

Reply:
<img width="700" alt="image" src="https://user-images.githubusercontent.com/894884/216476288-24b1c500-3938-469d-a404-13ff56b64565.png">

it's a minor thing really, but it stands out - the lines inside the pattern match body should have 4 spaces of leading whitespace